crypto/libsodium-fork/src/libsodium/crypto_vrf/ietfdraft03/convert.c

```diff
@@ -106,6 +114,26 @@ _vrf_ietfdraft03_hash_points(unsigned char c[16], const ge25519_p3 *P1,
     sodium_memzero(c1, 64);
 }
 
+/* Same as _vrf_ietfdraft03_hash_points but for ge25519_p2 points instead
+ * for the last two points */
```

Comment:
this function changes the functional signature from p3 to p2 and replaces point_to_string with the relevant new call to point_p2_to_string, otherwise it's as above